client/my-sites/plans/current-plan/main.jsx

```diff
@@ -48,9 +56,7 @@ const PlanDetailsComponent = React.createClass( {
 	render: function() {
 		const { selectedSite } = this.props;
 		const { hasLoadedFromServer } = this.props.sitePlans;
-		let title;
-		let tagLine;
-		let featuresList;
+		let currentPlan, title, tagLine, featuresList;
```

Comment:
If I'm reading this correctly, we should be able to simplify by doing:
```javascript
const currentPlan = this.props.selectedSite.plan.product_slug;
```
And then passing currentPlan into the `PlanIcon` component. The planIcon component already has a switch to handle all known plans.